src/frontend/src/main/SearchField/index.jsx

```diff
@@ -97,7 +97,7 @@ export default function SearchField({
         case 200:
           console.log('open successfully');
           break;
-        case 404:
+        case 204:
```

Comment:
[404 Not Found](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404)
The server cannot find the requested resource. In the browser, this means the URL is not recognized. In an API, this can also mean that the endpoint is valid but the resource itself does not exist. Servers may also send this response instead of 403 Forbidden to hide the existence of a resource from an unauthorized client. This response code is probably the most well known due to its frequent occurrence on the web.

[204 No Content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204)
There is no content to send for this request, but the headers may be useful. The user agent may update its cached headers for this resource with the new ones.

why have you changed this from 404 to 204? just need explanation. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status